mods/pokebattler.php

```diff
@@ -109,96 +109,55 @@
-                $pokemon = str_replace('_FORM', '', $raid['pokemon']);
-
-                // Get pokemon name and form.
-                $name = explode("_", $pokemon, 2)[0];
-                $form = explode("_", $pokemon, 2)[1];
-
-                // Fix for MEWTWO_A_FORM
-                if($name == 'MEWTWO' && $form == 'A') {
-                    $form = 'ARMORED';
-                }
-
-            // Pokemon name ending with "_MALE" ?
-            } else if(substr_compare($raid['pokemon'], '_MALE', -strlen('_MALE')) === 0) {
-                debug_log('Pokemon with gender MALE received: ' . $raid['pokemon']);
-                // Remove "_MALE"
-                $pokemon = str_replace('_MALE', '', $raid['pokemon']);
-
-                // Get pokemon name and form.
-                $name = explode("_", $pokemon, 2)[0] . '♂';
-                $form = 'normal';
-
-            // Pokemon name ending with "_FEMALE" ?
-            } else if(substr_compare($raid['pokemon'], '_FEMALE', -strlen('_FEMALE')) === 0) {
-                debug_log('Pokemon with gender FEMALE received: ' . $raid['pokemon']);
-                // Remove "_FEMALE"
-                $pokemon = str_replace('_FEMALE', '', $raid['pokemon']);
-
-                // Get pokemon name and form.
-                $name = explode("_", $pokemon, 2)[0] . '♀';
-                $form = 'normal';
-
-            // Mega pokemon ?
-            }else if(substr_compare($raid['pokemon'], '_MEGA', -strlen('_MEGA')) === 0 or substr_compare($raid['pokemon'], '_MEGA_X', -strlen('_MEGA_X')) === 0 or substr_compare($raid['pokemon'], '_MEGA_Y', -strlen('_MEGA_Y')) === 0) {
-                debug_log('Mega Pokemon received: ' . $raid['pokemon']);
-
-                // Get pokemon name and form.
-                $name_form = explode("_", $raid['pokemon'], 2);
-                $name = $name_form[0];
-                $form = $name_form[1];
-
-            // Normal pokemon without form or gender.
-            } else {
-                // Fix pokemon like "HO_OH"...
-                if(substr_count($raid['pokemon'], '_') >= 1) {
-                    $pokemon = str_replace('_', '-', $raid['pokemon']);
-                } else {
-                    $pokemon = $raid['pokemon'];
-                }
-                // Name and form.
-                $name = $pokemon;
-                $form = 'normal';
-
-		// Fix for GIRATINA as the actual GIRATINA_ALTERED_FORM is just GIRATINA
-                if($name == 'GIRATINA' && $form == 'normal') {
-                    $form = 'ALTERED';
-                }
-            }
-            // Get ID and form name used internally.
-            debug_log('Getting dex id and form for pokemon ' . $name . ' with form ' . $form);
-            $dex_id_form = get_pokemon_id_by_name($name . '-' . $form, true);
+            $dex_id_form = resolve_boss_name_to_ids($raid['pokemon']);
+            $bosses[$raid_level_id][] = $dex_id_form;
+        }
+    }
+    info_log(print_r($bosses,true));
```

Comment:
Should probably become debug_log? Or is it readable enough that it makes sense as info?

Reply:
Oops, forgot to delete that line. I don't think we need that info.